app/jobs/event_notification_job.rb

```diff
@@ -0,0 +1,20 @@
+class EventNotificationJob < ApplicationJob
+  def self.perform
+    presenter = {
+      first_name: 'Dennis',
+      last_name: 'Marchand',
+      email: 'dennis@urbantherapy.ca'
+    }
+
+    # Event.requiring_notification.each do |event|
```

Comment:
I think we should remove the commented out code. Instead, I'd prefer to add a `#TODO` comment to `presenter` above, detail that this is a stubbed model for now.